build.yaml

```diff
@@ -781,6 +806,7 @@ steps:
       mountPath: /test-gsa-key
    dependsOn:
     - default_ns
+    - upload_test_resources_to_gcs
```

Comment:
The other reason I prefer this to a global token is that getting the token requires you to write down the necessary dependency.